### PR TITLE
SH-108, SH-143, little improvements

### PR DESCRIPTION
**Pull request by sherzinger**

### CODE REVIEW

File: grails-app/views/layouts/_heatmap.gsp [hunk 1]
```diff
@@ -6,18 +6,39 @@
...
+                    <td>
+                        <concept-box concept-group="fetch.conceptBoxes.highDimensional"
+                                     type="HD"
+                                     min="1"
+                                     max="-1"
+                                     label="High Dimensional Variables"
+                                     tooltip="Select high dimensional data node(s) from the Data Set Explorer Tree and drag it into the box. The nodes needs to be from the same platform.">
+                        </concept-box>
+                    </td>
+                    %{--<td>--}%
```
Comment (kretep) [inline grails-app/views/layouts/_heatmap.gsp:21]: Is there a reason this commented-out code is still here? If it is not used for anything, I recommend deleting it.

Comment (sherzinger) [inline grails-app/views/layouts/_heatmap.gsp:21]: The code is functional, but the R side needs to be modified. This is part of the re-introduction of the LDD feature

Comment (kretep) [inline grails-app/views/layouts/_heatmap.gsp:21]: You mean it will be uncommented later?

Comment (sherzinger) [inline grails-app/views/layouts/_heatmap.gsp:21]: positive

**Pull request merged**